src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Hooks.java

```diff
@@ -17,6 +17,8 @@
 
     @Before
     public void openBrowser() {
+        String exePath = "C:\\Users\\amore\\eclipse-workspace\\chpl-testautomation\\chromedriver_win32\\chromedriver.exe";
```

Comment:
This will break any other user's environment, as well as Jenkins. You should have your chromedriver directory in your $PATH or %PATH% environment variable and remove this stuff